server/src/main/java/org/elasticsearch/search/internal/ShardSearchRequest.java

```diff
@@ -130,7 +130,7 @@ public ShardSearchRequest(ShardId shardId,
                               long nowInMillis,
                               AliasFilter aliasFilter) {
         this(OriginalIndices.NONE, shardId, -1, -1, SearchType.QUERY_THEN_FETCH, null, null,
-            aliasFilter, 1.0f, false, null, nowInMillis, null, null, null);
+            aliasFilter, 1.0f, true, null, nowInMillis, null, null, null);
```

Comment:
What's the reason of setting `allowPartialSearchResults` to `true` here?

I've checked and this constructor besides tests is used only in validate, explain and field capabilities actions.  I guess it is fine to return partial results here.

Reply:
Because the default value is true so it should be reflected here too.